src/panel/download_card/downloadActions.js

```diff
@@ -11,10 +11,7 @@ module.exports = {
 };
 
 function saveDownloadList(title, id) {
-	if (commonActions.isInvalidTitle(title) ||
-		commonActions.isDuplicateTitle(title) ||

```

Comment:
How come the title checks also removed? Are those checks happening elsewhere now?